Resources/public/css/views/serverside.css

```diff
@@ -2,7 +2,97 @@
  * Copyright (C) eZ Systems AS. All rights reserved.
  * For full copyright and license information view LICENSE file distributed with this source code.
  */
-
 .ez-view-serversideview .ez-platform-version {
     margin-top: 0;
 }
+
+.ez-view-serversideview .ez-serverside-content {
+    padding-top: 0px;
+}
+
+.ez-view-serversideview .ez-serverside-content-inner {
+    padding: 0.5em 1em;
+}
+
+.ez-view-serversideview .ez-serverside-content-inner > section > section {
+    padding-left: 2.5em;
+}
+
+.ez-view-serversideview .ez-serverside-content-inner .ez-section-title-name {
+    padding: 0.8em 0;
+}
+
+.ez-view-serversideview .ez-serverside-content-inner .ez-section-subtitle-name {
+    padding: 0.5em 0;
+}
+
+.ez-view-serversideview .ez-serverside-content-inner .ez-contenttype-metadata {
+    display: flex;
+}
+
+.ez-view-serversideview .ez-serverside-content-inner .ez-contenttype-metadata > * {
```

Comment:
we try to avoid the `*` selector, since ez-contenttype-metadata is supposed to contain ez-contenttype-metadata-name and ez-contenttype-metadata-value, this one is easy to expand :)